src/System.Runtime.Numerics/tests/BigInteger/MyBigInt.cs

```diff
@@ -791,7 +874,31 @@ private static List<byte> GetBytes(BitArray ba)
 
             return mask;
         }
-        private static String Print(byte[] bytes)
+
+        public static String Print(byte[] bytes)
+        {
+            String ret = "make ";
+
+            for (int i = 0; i < bytes.Length; i++)
+            {
+                ret += bytes[i] + " ";
```

Comment:
To handle the empty edge case, the statement would have to be:
`"make " + (bytes.Length > 0 ? string.Join(" ", bytes) + " " : string.Empty) + "endmake ";`
I don't think this is better than what exists.